housekeep.py

```diff
@@ -144,26 +175,64 @@ def dispatch_superclean():
 
 
 def get_opts(argv):
+    # From: https://stackoverflow.com/a/49999185/149900
+    class NoAction(argparse.Action):
+        def __init__(self, **kwargs):
+            kwargs.setdefault("default", argparse.SUPPRESS)
+            kwargs.setdefault("nargs", 0)
+            super().__init__(**kwargs)
+
+        def __call__(self, *args, **kwargs):
+            pass
+
+    dispers = {
+        name.replace("dispatch_", ""): inspect.getdoc(obj)
+        for name, obj in inspect.getmembers(sys.modules[__name__])
+        if name.startswith("dispatch_") and inspect.isfunction(obj)
+    }
+
     parser = argparse.ArgumentParser()
+    parser.register("action", "no_action", NoAction)
+
     parser.add_argument(
-        "cmd", metavar="CMD", choices=["setup", "cleanup", "superclean"]
+        "--force", "-F", action="store_true", help="Force action even if in CI"
     )
+    parser.add_argument(
+        "cmd", metavar="COMMAND", choices=sorted(dispers.keys()), help="(See below)"
+    )
+
+    cgrp = parser.add_argument_group(title="COMMAND is one of")
+    for name, doc in sorted(dispers.items()):
+        cgrp.add_argument(name, help=doc, action="no_action")
+
     return parser.parse_args(argv)
 
 
 def python_interp_details():
-    print(f"{FG_CYAN}Python interpreter details:{NORM}")
+    print(f"{Fore.CYAN}Python interpreter details:")
     details = sys.version.splitlines() + sys.executable.splitlines()
     for ln in details:
-        print(f"    {FG_CYAN}{ln}{NORM}")
+        print(f"    {Fore.CYAN}{ln}")
+    print(Style.RESET_ALL, end="", flush=True)
 
 
 if __name__ == "__main__":
+    colorama_init is None or colorama_init(autoreset=True)
     python_interp_details()
-    if os.environ.get("CI") == "true":
-        # All the housekeeping steps are pointless on Travis CI / GitHub Actions;
-        # they build and tear down their VMs everytime anyways.
-        sys.exit(0)
     opts = get_opts(sys.argv[1:])
+    if os.environ.get("CI") == "true":
+        if not opts.force:
```

Comment:
To cater people who has their own CI for testing this library and still want all the goodies `housekeep.py` provides.